### PR TITLE
Update the steps to create an azure service principal

### PR DESCRIPTION
**Pull request by nader-ziada**
adjust the instructions based on changes in the az cli now using `az ad sp create-for-rbac` instead of `sp create`

[#184740139]

### CODE REVIEW

**Comment by anita-flegg** [comment]
@nader-ziada , Does this apply to both 3.0 and 2.10 or is this a draft PR for the upcoming release?
(Or maybe it really is just for 3.0 :)

**Comment by nader-ziada** [comment]
it applies for both but I created another one for 2.10, let me know if that is not needed. Thanks

**Review by anita-flegg — CHANGES_REQUESTED** [review]
Please ensure that the Step numbering in the level headings (##) are still sequential.
It looks like you have deleted Step 4 without updating the rest of the Step umber headings.

File: azure/prepare-azure-terraform.html.md.erb [hunk 1]
```diff
@@ -142,70 +141,38 @@ deploy <%= vars.ops_manager %> in order to deploy <%= vars.app_runtime_first %>
     }
     </pre>
 
+## <a id='create-configure'></a> Step 4: Create and configure a service principal
 
-## <a id='create-credential'></a> Step 4: Create a credential for the ADD app
```
[the remark this answers is not in the document]
Comment (nader-ziada) [reply azure/prepare-azure-terraform.html.md.erb:146]: fixes the numbers of the following steps

**Pull request merged**